ui/app/controllers/allocations/allocation/index.js

```diff
@@ -85,7 +86,7 @@ export default class IndexController extends Controller.extend(Sortable) {
     } catch (err) {
       this.set('error', {
         title: 'Could Not Restart Allocation',
-        description: 'Your ACL token does not grant allocation lifecycle permissions.',
+        description: messageForError(err, 'manage allocation life cycle'),
```

Comment:
The [documentation](https://www.nomadproject.io/docs/job-specification/lifecycle) seems to consistently use `lifecycle` vs `life cycle`, should this too?

Reply:
Good call.